skycoin/skywire
# Health checks for services

Did you run `make format && make check`? Yes

Fixes #422

 Changes:	
- Health checks for services
- One address resolver client for all transports
- Add uptime tracker and address resolver statuses to frontend

How to test this PR:
- Run generic env
- Go to the hypervisor UI
- Check if transport discovery, route finder, setup node are up and if uptime tracker, address resolver are up/down depending on the config
